build_dockers.yml

```diff
@@ -80,6 +80,14 @@
         - name: Set package subdir as fact
           set_fact: _package_subdir={{ _ls_register.stdout }}
 
+        - name: Build spark-base image locally
+          shell: >
+            docker build --no-cache  \
+                -t spark-base:latest  \
```

Comment:
Wondering, if we can use same docker_tag as the driver/executor dockers, so there are no accidental rewrites of spark-base images during concurrent runs as "latest" tag will be shared among them.